WootricSDK/WootricSDK/WTRSettings.m

```diff
@@ -94,6 +109,34 @@ - (void)parseDataFromSurveyServer:(NSDictionary *)surveyServerSettings {
   }
 }
 
+- (NSDictionary *)scoreRules {
+    return @{
+             @"NPS" : [NSArray arrayWithObjects: @{@"min" : [NSNumber numberWithInt:0], @"max" : [NSNumber numberWithInt:10], @"negative_type_max" : [NSNumber numberWithInt:6], @"neutral_type_max" : [NSNumber numberWithInt:8]}, nil],
```

Comment:
Why are you creating an array? It's pointless, just have the dictionary inside the dictionary.